### DISCUSSION TITLE
Another error, type mismatch

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#2058] [Q&A]
```
Progress  5.62it/s ████▊                               14% 5/35 00:00 00:05 Base18:16:13-163747 ERROR    Error setting current image: step=5 Input type (struct c10::Half) and bias type (float) should be the same

Progress  5.61it/s ██████████▎                        31% 11/35 00:01 00:04 Base18:16:14-181224 ERROR    Error setting current image: step=11 Input type (struct c10::Half) and bias type (float) should be the same

Progress  5.64it/s ███████████████                    46% 16/35 00:02 00:03 Base18:16:15-188559 ERROR    Error setting current image: step=16 Input type (struct c10::Half) and bias type (float) should be the same

Progress  5.61it/s █████████████████████▋             66% 23/35 00:04 00:02 Base18:16:16-296026 ERROR    Error setting current image: step=23 Input type (struct c10::Half) and bias type (float) should be the same

Progress  5.62it/s ██████████████████████████▍        80% 28/35 00:05 00:01 Base18:16:17-320970 ERROR    Error setting current image: step=28 Input type (struct c10::Half) and bias type (float) should be the same

Progress  5.64it/s ████████████████████████████████   97% 34/35 00:06 00:00 Base18:16:18-339462 ERROR    Error setting current image: step=34 Input type (struct c10::Half) and bias type (float) should be the same

Progress  5.60it/s █████████████████████████████████ 100% 35/35 00:06 00:00 Base
18:16:19-354663 ERROR    Error setting current image: step=35 Input type (struct c10::Half) and bias type (float) should be the same
18:16:20-388703 ERROR    Error setting current image: step=35 Input type (struct c10::Half) and bias type (float) should be the same
```

This pops up in the log every few seconds while I'm actively generating images with an SDXL model (base or other from CivitAI). It doesn't stop images from generating. The UI gives no indication of a problem.

I'm running an installation of SDNext dedicated to playing with SDXL, so I only have a small set of XL models and one user-installed extension (dynamic-prompts, not activated).

**Comment by vladmandic** [comment]
that type of an error is always platform/gpu and settings related - and you've provided no information.

**Comment by afbagwell** [comment]
Settings images:
<img width="1707" alt="Screenshot 2023-08-23 002909" src="https://github.com/vladmandic/automatic/assets/1495201/a440f76e-55ad-4f18-ad62-0b1990dea6c2">
<img width="616" alt="Screenshot 2023-08-23 003057" src="https://github.com/vladmandic/automatic/assets/1495201/5ae39098-89e8-4be4-98c6-df15f369ab49">

**Comment by vladmandic** [comment]
error is happening during generation of live preview image, not during actual main processing - that's why it doesn't abort.
what are your live preview settings? try settings taesd as preview type if its not already set.

**Comment by afbagwell** [comment]
Ah yes, moving to TAESD fixed the problem. It's interesting because I duplicated my live preview settings from my 1.5 model instance to the SDXL one, and the former uses "Full VAE" without throwing the same error.

**Reply by vladmandic** [reply]
I'll fix full vae, so let's keep this open.
But in general full vae is really resource intensive, definitely not recommended for live preview.
Full vae is only useful if you want to capture every frame for animation purposes.

**Reply by afbagwell** [reply]
Good to know. Thanks!
